src/main/java/com/jpop/productservice/controller/ProductServiceController.java

```diff
@@ -35,11 +38,23 @@ public ProductServiceController(ProductInsertService productInsertService, Produ
         this.productDeleteService = productDeleteService;
     }
 
+    @ApiOperation(value = "Get all products", notes = "This will be used to get all the products in inventory")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Product found in inventory"),
+            @ApiResponse(code = 404, message = "URL not found"),
+            @ApiResponse(code = 400, message = "Bad request")
```

Comment:
API clients use swagger documentation to code accordingly. So we need to give the API Responses that we have actually coded. In `getAllProducts` case, I don't think we return 404, 400 codes at any point. Please remove them.

This comment applies to all other operations in this API

Reply:
Yes agree, I have changed the same.